### DISCUSSION TITLE
FieldValues

### DISCUSSION THREAD

**Discussion by mdigger** [redis/rueidis#133] [Ideas]
I understand that making incompatible changes to the API is bad! :) 
Especially if it is done for the sake of beauty. 
But there is a lack of plural for `FieldValue` when bilding a command:

```golang
cmd := c.client.B().
	Hset().
	Key("key:name").
	FieldValues(). // <- *
	FieldValue("id", Id).
	FieldValue("name", name).
	FieldValue("disabled", strconv.FormatBool(disabled)).
	Build()
```

**Comment by rueian** [comment]
Good idea. I think adding FieldValues() will not break the compatibility.
